k8s/showcase/vineyard-mars-pytorch/Dockerfile.mars-with-vineyard

```diff
@@ -0,0 +1,8 @@
+# TODO: wait for the pr (https://github.com/mars-project/mars/pull/3324) released
+FROM marsproject/mars:v0.10.0(use the next version here)
+
+WORKDIR /srv
+
+RUN pip3 install vineyard==0.11.7
```

Comment:
No need to fix the version, I think.

Reply:
Okay, so we don't need to expose the dockerfile to users?